Rakefile

```diff
@@ -1,37 +1,29 @@
 require 'rake'
 require 'rake/testtask'
 require 'robot-controller/tasks'
-begin
-  require 'rubocop/rake_task'
-  RuboCop::RakeTask.new(:rubocop) do |task|
-   task.options = ['-l'] # run lint cops only
-  end
-rescue LoadError
-  task :rubocop do
-    puts "Rubocop is not installed"
-  end
-end
+require 'rubocop/rake_task'
```

Comment:
Will removing the `begin`/`rescue` block cause problems running rake tasks in production, because `rubocop` is not available as a dependency? (Not sure we need to run rake tasks in prod, though.)

Reply:
I haven't seen this cause trouble elsewhere. I'll keep an eye on it.